smart_campus/app/templates/app/reward_edit_page.html

```diff
@@ -0,0 +1,94 @@
+{% extends 'app/base.html' %}
+{% load static %}
+{% block stylesheet %}
+  {{ block.super }}
+{% endblock %}
+<style rel="stylesheet" type="text/css">
```

Comment:
Is there any reason why this code block cannot be written in its CSS file?

Reply:
No, I think it just a typo.